test/integration/plugin/tests/test-plugin-resource-launch.hpp

```diff
@@ -41,7 +41,7 @@ void PluginResourceLaunchTestImpl()
 
       RAJA::launch<LaunchPolicy>
         (res, RAJA::LaunchParams(RAJA::Teams(1), RAJA::Threads(1)),
-         [=] RAJA_HOST_DEVICE(RAJA::LaunchContext ctx)
+         [=] RAJA_HOST_DEVICE(RAJA::LaunchContext)
```

Comment:
I think it better to use the RAJA_UNUSED_ARG macro for this since it calls attention to it, can be easily searched for in the code, and is more consistent with what we do in other places.